benchmark/B-FHTL/README.md

```diff
@@ -2,23 +2,24 @@
 
 B-FHTL is a benchmark for federated hetero-task learning. 
 It contains three federated datasets: Graph-DC, Graph-DT and Text-DT. 
-Here we provide the datasets and scripts of the baselines.
+Here we provide the datasets and scripts for the baselines.
 More details about the benchmark please refer to [https://arxiv.org/abs/2206.03436](https://arxiv.org/abs/2206.03436)
 
 **Notice**:
-Considering FederatedScope is an open-sourced library that updates frequently, to ensure the reproducibility of the experimental results, we create a new branch `feature/B-FHTL`. The users can reproduce the results by running the scripts under the directory `scripts/`, you can also copy the scripts from `benchmark/B-FHTL/${DATA_NAME}`.  
+Considering FederatedScope is an open-sourced library that updates frequently, to ensure the reproducibility of the experimental results, 
+we create a new branch `feature/B-FHTL`. The users can reproduce the results by running the scripts under the directory [`scripts/B-FHTL_exp_scripts/`](https://github.com/alibaba/FederatedScope/tree/feature/B-FHTL/scripts/B-FHTL_exp_scripts), 
+you can also copy the scripts from `benchmark/B-FHTL/${DATA_NAME}` into your own directory.  
 
 ## Dataset
 ### Graph-DC
 The dataset Graph-DC is consisted of 13 graph classification tasks. Each task represents a client in federated learning, and hold different learning goals. You can choose this dataset by setting `data.type` as `fs_contest_data`.
 
 ### Graph-DT
-The dataset Graph-DT is consisted of 17 graph tasks, including 10 binary classification tasks, 1 multi-class classification task and 5 regression tasks. You can choose this dataset by setting `data.type` as ``.
-
+The dataset Graph-DT is consisted of 17 graph tasks, including 10 binary classification tasks, 1 multi-class classification task and 5 regression tasks. You can choose this dataset by setting `data.type` as `graph-dt`.
```

Comment:
The dataset Graph-DT is consisted of 16 graph tasks, including 10 binary classification tasks, 6 regression tasks.

Reply:
It is modified accordingly.